client/src/components/client-dashboard-component/ClientTable.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from "prop-types";
+import { DataGrid } from "@mui/x-data-grid";
+import { Box, Card, Link, Typography, Avatar } from "@mui/material";
+
+const columns = [
+  {
+    field: "name",
+    headerName: "Name",
+    width: 175,
+    renderCell: (params) => (
+      <Link href={`/clients/${params.id}`} underline="hover">
+        {params.value}
+      </Link>
+    ),
+  },
+  {
+    field: "phone",
+    headerName: "Phone Number",
+    width: 175,
+  },
+  {
+    field: "email",
+    headerName: "Email",
+    width: 175,
+    renderCell: (params) => (
+      <Link href={`mailto:${params.value}`} underline="hover">
+        {params.value}
+      </Link>
+    ),
+  },
+  {
+    field: "status",
+    headerName: "Status",
+    width: 175,
+    renderCell: (params) => (
+      <Box
+        sx={{
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+          backgroundColor: "#ececec",
+          borderRadius: "30%",
+          width: "fit-content",
+          padding: "3px 8px",
+          fontWeight: "bold",
+        }}
+      >
+        <Typography variant="body2" component="span">
+          {params.value}
+        </Typography>
+      </Box>
+    ),
+  },
+  {
+    field: "dateUpdated",
+    headerName: "Date Updated",
+    width: 175,
+  },
+  {
+    field: "owner",
+    headerName: "Owner",
+    width: 175,
+    renderCell: (params) => {
+      const getInitials = (name) => {
+        return name
+          .split(" ")
+          .map((n) => n[0])
+          .join("");
+      };
+
+      return (
+        <Box
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            gap: 1,
+            backgroundColor: "#ececec",
+            borderRadius: "30%",
+            width: "fit-content",
+            padding: "4px 4px",
+          }}
+        >
+          <Avatar sx={{ width: 22, height: 22, fontSize: "12px" }}>
+            {getInitials(params.value)}
+          </Avatar>
+          <Box>{params.value}</Box>
+        </Box>
+      );
+    },
+  },
+];
+
+function ClientTable({
+  clientData,
+  paginationModel,
+  setPaginationModel,
+  totalRowCount,
+}) {
+
+
+  return (
```

Comment:
atm when a client hovers over the title of a column or clicks on one there seems to be some indication of ordering present (up arrow or down arrow) beside the title of that column. I think Kevin's job leads table doesn't have that so you can definitely use it as a reference for removing this. <img width="1057" alt="Screenshot 2024-01-26 at 12 41 55 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/95ffe1b8-0c26-4b42-84c2-3d18ee18894a">